lib/src/repository/devices/devices_repository.dart

```diff
@@ -128,14 +128,14 @@ class DevicesRepository {
     if (trait is GgetDeviceData_device_traits__asThermostatSettingDeviceTrait ||
         trait
             is GgetDevicesData_me_devices_edges_node_traits__asThermostatSettingDeviceTrait) {
-      final properties = new Set<Property>.from(trait
+      final availableFanMode = new Set<AvailableFanMode>.from(trait
           .properties.availableFanModes
           .map((mode) => AvailableFanMode((mode as GFanMode).name)));
```

Comment:
Interesting, this way we can build an Enum directly from the generated code / schema. In the future maybe we can write a common util library for this.